easy/strings/20.ValidParentheses.js

```diff
@@ -1,26 +0,0 @@
-function isValid(string) {
```

Comment:
Please don't use `string` as a variable name, even if it's allowed in JS.
In most languages, it's the reserved keyword for a type name. 
Perhaps `input` instead?